spec/v3/services/enterprise_license/find_spec.rb

```diff
@@ -0,0 +1,34 @@
+describe Travis::API::V3::Services::EnterpriseLicense::Find, set_app: true do
+  let(:parsed_body) { JSON.load(body) }
+
+  before do
+
+    user = Factory(:user, emails: [Email.new(email: "hello@example.com")])
+    user1 = Factory(:user, emails: [Email.new(email: "hello2@example.com")])
+    user2 = Factory(:user, emails: [Email.new(email: "hello3@example.com")])
+    user3 = Factory(:user, emails: [Email.new(email: "hello4@example.com")])
+
+    Factory(:commit, committer_email: user.emails.first.email)
+    Factory(:commit, committer_email: user1.emails.first.email)
+    Factory(:commit, committer_email: user2.emails.first.email)
+    Factory(:commit, committer_email: user2.emails.first.email)
+    Factory(:commit, created_at: "2017-05-12 08:43:23", committer_email: user3.emails.first.email)
+
+    replicated_endpoint = 'https://10.169.183.13:9880'
```

Comment:
Is this IP anything in particular or just something specific for the stub request to work? In the case of the latter something like a `https://fake..fakeserver.com:9880` may be more communicative. Or just a comment. Probably comments either way to make it clear we are or are not using an external server here for the tests.